### ISSUE TITLE
Performance post Fine-Tuning

### ISSUE THREAD

**Issue by rraina97** [tesseract-ocr/tessdata_best#53]
Used ara.traineddata from tessdata_best directory and finetuned it using approximately 300 arabic numerical data. Though the accuracy on arabic numerals has increased, but the tuned model performs drastically bad than the original model. Any steps to avoid such error ?
